examples/flutter_gallery/lib/demo/calculator/logic.dart

```diff
@@ -258,6 +260,7 @@ class CalcExpression {
       case ExpressionState.Result:
         return appendOperation(Operation.Subtraction);
     }
+    return null;
```

Comment:
I'd make this one a `default:` case, i think, to make it more obvious it was intended to catch the other cases. Actually I'm surprised that this compiles, I guess maybe we currently don't have any cases where you can't append a `-` ?